### ISSUE TITLE
xml2acb

### ISSUE THREAD

**Issue by urabbit** [atesgoral/acb#36]
Is there a program that can achieve xml2acb？

**Comment by atesgoral** [comment]
@urabbit can you elaborate?

**Comment by urabbit** [comment]
@atesgoral Yes, I downloaded the XML2ACB program and can export  XML. I hope I can edit the XML and then generate ACB

**Comment by atesgoral** [comment]
I unfortunately never wrote an xml2acb. If you can use this JavaScript acb library, you wouldn't need to switch to XML at all.
